runners/direct-java/pom.xml

```diff
@@ -74,6 +74,9 @@
                 <dependency>org.apache.beam:beam-sdks-java-core</dependency>
                 <dependency>org.apache.beam:beam-runners-core-java</dependency>
               </dependenciesToScan>
+              <excludes>
+                <exclude>org.apache.beam.sdk.transforms.OldDoFnTest</exclude>
```

Comment:
Done. I think it all works now without exclusion. The inherited `aggregators` field continues to be a bit of a pain.